packages/web-components/@sfx/sayt/test/unit/sayt.test.ts

```diff
@@ -267,6 +280,30 @@ describe('Sayt Component', () => {
     });
   });
 
+  describe('debounceSaytRequests', () => {
+    it('should debounce the requestSaytAutocompleteTerms and requestSaytProducts methods', () => {
+      const delay = sayt.debounceTime;
+      const debounceSettings = { 'leading': true, 'trailing': true };
+      const debounceStub = stub(Lodash, 'debounce');
+      const debounceSaytRequests = stub(sayt, 'debounceSaytRequests');
+
+      const requestSaytAutocompleteTerms = debounceStub(sayt.requestSaytAutocompleteTerms(), delay, debounceSettings);
+      const requestSaytProducts = debounceStub(sayt.requestSaytProducts(), delay, debounceSettings);
+      const autocompleteTermsReturn = debounceSaytRequests.returns(requestSaytAutocompleteTerms);
+      const productsReturn = debounceSaytRequests.returns(requestSaytProducts);
+
+      sayt.debounceSaytRequests(sayt.requestSaytAutocompleteTerms);
+      sayt.debounceSaytRequests(sayt.requestSaytProducts);
+
+      expect(debounceStub).to.be.calledTwice;
+      expect(debounceSaytRequests).to.be.calledTwice;
+      expect(debounceStub).to.be.calledWithExactly(requestSaytAutocompleteTerms, delay, debounceSettings);
+      expect(debounceStub).to.be.calledWithExactly(requestSaytProducts, delay, debounceSettings);
+      expect(debounceSaytRequests).to.equal(autocompleteTermsReturn);
+      expect(debounceSaytRequests).to.equal(productsReturn);
```

Comment:
I don't believe we're asserting anything of value on the above two lines. I'm curious what your thought process is, but as far as I can tell, you're actually defining these things to be equal because called `.returns()` on a stub returns the stub itself. So you define `debounceSaytRequests` to be a stub, and then later make a new variable equal to `debounceSaytRequests.returns(...)`, so that new variable is definitionally identical to `debounceSaytRequests`.